lib/internal/fs/streams.js

```diff
@@ -340,7 +327,6 @@ WriteStream.prototype._writev = function(data, cb) {
 
   fs.writev(this.fd, chunks, this.pos, function(er, bytes) {
     if (er) {
-      self.destroy();
```

Comment:
This was a bug where the stream would be destroyed even though autoClose might be disabled.